src/helpers/auth.js

```diff
@@ -0,0 +1,17 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+
+dotenv.config();
+
+/**
+ * @method GenToken
```

Comment:
this is a function and not a class method. Also I the name here should be the exact match of the function name